integtest/iceberg_real_hsi_test.py

```diff
@@ -0,0 +1,203 @@
+import pytest
+import os
+import re
+import psutil
+import copy
+import urllib.request
+
+import dfmodules.data_file_checks as data_file_checks
```

Comment:
`hsilibs` does not depend on `dfmodules`, which means that this may not necessarily be available. We may want to move these utility functions to a more central location...